js/views/reuse.vue

```diff
@@ -38,8 +38,14 @@ import Metrics from 'models/metrics';
 import Vue from 'vue';
 import Issues from 'models/issues';
 import Discussions from 'models/discussions';
+// Widgets
+import DatasetCards from 'components/dataset/card-list.vue';
+import DiscussionList from 'components/discussions/list.vue';
+import IssueList from 'components/issues/list.vue';
 import Layout from 'components/layout.vue';
 
+const MASK = `datasets{${DatasetCards.MASK}},*`;
```

Comment:
I've chosen the shorter version: required dataset fields and all reuse fields (not so many bu most of them used on the page). The biggest data was the datasets payload (and all cards are now fetch once with only the required fields vs one call by card before with all fields)